sips/minutes/_posts/2016-11-29-sip-minutes.md

```diff
@@ -0,0 +1,97 @@
+
+
+## Date and Location
+The meeting took place on 29 November 2016 via Google Hangouts at EPFL in Lausanne, Switzerland as well as other locations.
+
+Minutes were taken by Travis Lee.
+
+## Attendees
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Jorge Vicente Cantero ([@jvican](https://github.com/jvican)), Process Lead
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+* Heather Miller ([@heathermiller](https://github.com/heathermiller)), Scala Center
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), EPFL
+* Eugene Burmako ([@xeno-by](https://github.com/xeno-by)), EPFL
+* Iulian Dragos ([@dragos](https://github.com/dragos)), Independent
+* Adriaan Moors ([@adriaanm](https://github.com/adriaanm)), Lightbend
+* Dale Wijnand ([@dwijnand](https://github.com/dwijnand)), author of SIP-27
+
+
+## Proceedings
+### Opening Remarks
+
+**Jorge** We'll talk about the SIPS for META. Eugene will start.
+
+### [SIP-28 and SIP-29 - Inline meta](http://docs.scala-lang.org/sips/pending/inline-meta.html)
+
+Eugene and co have been working hard for two months on inline and META. Previously discussed new macro system with new inline and meta features. Inline provides a facility to declare methods with inline right hand side into call side (0:01:24) and meta implements compile-time function execution to do meta-programming. Martin implemented inline mechanism in Dotty. Eugene worked on macro notations. New style macros will integrate with tools. Eugene shows how it works in IntelliJ. For example, you can print the value of the parameters. Meta blocks supported by IntelliJ. So are quasi-quotes. You can also expand macros. Will greatly help debugability.
+
+Iulian says we should flesh out the Scala meta API.
+
+The spec needs to be updated based on Martin's Dotty implementation. We need to split the SIPs for the next meeting.
+
+**Conclusion** More development needed. Must answer questions on PR.
+
+### [SIP-24 - Repeated By Name Parameters](http://docs.scala-lang.org/sips/pending/repeated-byname.html)
+
+Heather says the debate is about the semantics or translation rules. All arguments are evaluated each time the parameter is referenced in the method. This is implemented in Dotty. Should this be implemented in Scalac?
+
+Sébastien thinks that Java bridge should be forbidden in this case. It's annotation that we add, so we could just make it an error.
+
+The SIP needs more examples. Martin says it's to remove an annoying restriction; there's no reason we can't we mix repeated and by name parameters. Evaluating them all together is much simpler. There's another reason regarding case classes. Inline function should maintain call by value semantics. Pass parameters by name just to be clean.
+
+The main motivation is to prepare for inline. Inline won't work very well without this. Need to flesh this out in SIP.
+
+**Conclusion** Someone needs to flesh out the SIP and how it relates to inline.
```

Comment:
There is not an implementation for Scalac, but for Dotty. This proposal is on hold until the Committee decides the specifics of the Inline proposal and how it relates to it. After that, the author will resubmit the proposal for further analysis.